repl/src/main/scala/ammonite/repl/Main.scala

```diff
@@ -71,21 +81,52 @@ case class Main(predef: String = "",
     res
   }
 
+
   /**
     * Run a Scala script file! takes the path to the file as well as an array
     * of `args` and a map of keyword `kwargs` to pass to that file.
     */
   def runScript(path: Path,
                 args: Seq[String],
-                kwargs: Map[String, String]): Res[Imports] = {
-
-    val repl = instantiateRepl()
+                kwargs: Map[String, String],
+                storage: Storage): Res[Seq[ImportData]] = {
+    val outBuffer = mutable.Buffer.empty[String]
+    val warningBuffer = mutable.Buffer.empty[String]
+    val errorBuffer = mutable.Buffer.empty[String]
+    val infoBuffer = mutable.Buffer.empty[String]
+    val prompt = Ref("@ ")
+    val colors = Ref[Colors](Colors.Default)
+    val frontEnd = Ref[FrontEnd](AmmoniteFrontEnd(Filter.empty))
+    val printer = Printer(
```

Comment:
This is probably incorrect; we don't print stuff much when running scripts, but anything we do print should be printed "normally" e.g. via https://github.com/lihaoyi/Ammonite/blob/master/repl/src/main/scala/ammonite/repl/Repl.scala#L31-L36, and not aggregated into a bunch of random buffers to be thrown away